lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -274,20 +276,22 @@ newStakePoolLayer db@DBLayer{..} nl tr = StakePoolLayer
 
         (distr, prod) <- liftIO . atomically $ (,)
             <$> (Map.fromList <$> readStakeDistribution nodeEpoch)
-            <*> (count <$> readPoolProduction nodeEpoch)
+            <*> readPoolProduction nodeEpoch
+
+        let tip = maximumBy (compare `on` view #slotId)
+                $ mconcat (Map.elems prod)
 
         when (Map.null distr || Map.null prod) $ do
-            computeProgress nodeTip >>= throwE . ErrMetricsIsUnsynced
+            computeProgress tip >>= throwE . ErrMetricsIsUnsynced
```

Comment:
I think here is a case when we actually don't want the `poolProductionTip` 😅 

`computeProgress` compares the `poolProductionTip` with this tip, so I think it will only display 100% now.

Reply:
I see two options:
- Keep it comparing with the node tip
- Compare against slot 0 of the node's epoch (should be when the results are actually first displayed 🤔 )

Maybe best to keep it the same for now.

Reply:
Done :+1: